ch-covidcertificate-backend-config/ch-covidcertificate-backend-config-wallet-ws/src/main/java/ch/admin/bag/covidcertificate/backend/config/wallet/ws/controller/WalletConfigController.java

```diff
@@ -145,7 +145,7 @@ public WalletConfigController(
         }
         for (var faqs: configResponse.getTransferWorks().values()) {
             faqs.getFaqIntroSections().forEach(
-                entry -> entry.setText(entry.getText().replaceAll("<transferCodeValidity>", transferCodeValidity)));
+                entry -> entry.setText(entry.getText().replaceAll("\\{TRANSFER_CODE_VALIDITY}", transferCodeValidity)));
```

Comment:
Shouldn't we escape both brackets? Just to be sure?

Reply:
IntelliJ complains that escaping the closing bracket is unnecessary. It works but I figured I'd stick to what the linter says

Reply:
use .replace() instead of replaceAll(). it doesnt make sense to use a regex here